mlfinlab/online_portfolio_selection/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
I just remembered - you need to expose the submodule in the `mlfinlab/__init__.py` file as well. There are examples in that file as well.

Reply:
Otherwise people wont be able to use the auto find when they type `mlfinlab.`

Reply:
I went through the other modules but couldn't find an example, are you saying that within the benchmarks directory, I make an __init__.py?

Reply:
No, this is the file I am referring to: https://github.com/hudson-and-thames/mlfinlab/blob/master/mlfinlab/__init__.py